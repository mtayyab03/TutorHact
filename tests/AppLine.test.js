import React from "react";
import { render } from "@testing-library/react-native";
import AppLine from "../app/components/AppLine";

it("renders correctly", () => {
  const tree = render(<AppLine />);
  expect(tree).toMatchSnapshot();
});
