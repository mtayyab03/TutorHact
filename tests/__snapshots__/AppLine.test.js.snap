// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#E8E8E8",
      "borderRadius": 7,
      "height": 1,
      "width": "100%",
    }
  }
/>
`;
